base/broadcast.jl

```diff
@@ -1244,22 +1225,21 @@ macro __dot__(x)
     esc(__dot__(x))
 end
 
-function Base.show(io::IO, bc::Broadcasted)
-    print(io, "Broadcasted(", bc.f)
-    args = bc.args
-    while args != TupleLLEnd()
-        print(io, ", ", args.head)
-        args = args.rest
-    end
-    print(io, ')')
-end
+Base.show(io::IO, bc::Broadcasted) = print(io, "Broadcasted(", bc.f, ", ", bc.args, ')')
```

Comment:
do we want the parens around the args? If not, could use `join(io, bc.args, ", ")` instead